content/docs/tasks/traffic-management/secure-ingress/index.md

```diff
@@ -466,6 +466,391 @@ only this time for host `bookinfo.com` instead of `httpbin.example.com`.
         `"""`
     {{< /text >}}
 
+## Configure a TLS ingress gateway with secret discovery service enabled
+
+In this section you will configure a TLS ingress gateway that fetches credentials from ingress gateway agent via secret discovery service (SDS).
+The ingress gateway agent is running in the same pod as the ingress gateway, and watches credentials created in the same namespace as the ingress gateway.
```

Comment:
Please list the advantages of using SDS:
1. Dynamic update of ingress gateway key/cert
2. Does not rely on local FS.
3. Supports watching multiple key/certs ...

Reply:
Done.